react/Typography/Readme.md

```diff
@@ -52,7 +52,7 @@ From old to new :
 ```jsx
 import { Text, Caption, MainTitle, SubTitle, Title, Bold, ErrorMessage } from 'cozy-ui/transpiled/react/Text'
 import MuiCozyTheme from 'cozy-ui/transpiled/react/MuiCozyTheme'
-import Typography from 'cozy-ui/transpiled/react/';
+import Typography from 'cozy-ui/transpiled/react/Typography';
```

Comment:
This one should not be in the `Do not change import for private method` commit

Reply:
it should be fixed now. I added it in the first commit